common-rendering/src/components/keyEvents.stories.tsx

```diff
@@ -68,26 +59,43 @@ const events: KeyEvent[] = [
 	},
 ];
 
-const Default = (args: { theme: Theme }) => (
-	<KeyEvents keyEvents={events} {...args} />
+const keyEventComp = (dark: boolean, theme: Theme, title: string) => (
+	<div css={flexItemStyles}>
+		<div>{title}</div>
+		<KeyEvents keyEvents={events} theme={theme} supportsDarkMode={dark} />
+	</div>
 );
 
-Default.args = {
-	theme: Pillar.News,
-};
+const flexItemStyles = css`
+	flex-grow: 1;
+`;
+
+const keyEventWithTheme = (dark: boolean) => () => (
+	<div
+		css={css`
+			display: flex;
+			flex-direction: row;
+			justify-content: space-between;
+			flex-wrap: wrap;
+			-ms-flex-positive: 1;
+		`}
+	>
+		{keyEventComp(dark, Pillar.News, "News")}
+		{keyEventComp(dark, Pillar.Culture, "Culture")}
+		{keyEventComp(dark, Pillar.Lifestyle, "Lifestyle")}
+		{keyEventComp(dark, Pillar.Opinion, "Opinion")}
+		{keyEventComp(dark, Pillar.Sport, "Sport")}
```

Comment:
I know we're not including `Labs` and `Special` themes here because the designs haven't been done for them but  we should probably support them being passed in because they are both valid values for `theme`. Maybe they can default to "News" while we wait for the colours to be worked out?

Reply:
They are actually defaulted to News within the `KeyEvents` component. So I just added the them to the storybook options.

Reply:
Ah, I see. Cool, cool, cool